apps/artemis-web/pages/signin.tsx

```diff
@@ -22,14 +17,18 @@ const SigninPage: React.FunctionComponent<{}> = () => {
         <title>ARTEMIS - Login</title>
       </Head>
       <div id="login-container">
-        {!user && !loading && (
+        {!user && (
           <div id="content-wrap" style={{ paddingBottom: '5rem' }}>
-            <SignInComponent />
+            <SignInComponent {...props} />
           </div>
         )}
       </div>
     </>
   );
 };
 
-export default SigninPage;
+// export async function getServerSideProps(ctx) {
+//   return getProps(ctx);
+// }
+
+export default withAuth(SigninPage, ['pending']);
```

Comment:
this means that signin page is only accessible for pending users which doesn't make sense